cpython/remove_modules.txt

```diff
@@ -1,12 +0,0 @@
-_osx_support.py
-curses
```

Comment:
Could you revert this change if you removed this file by a mistake?

Reply:
Whoops, was from when I was trying to get around #2449. Fixed.